CHANGES

```diff
@@ -2,14 +2,20 @@ Back In Time
 
 Version 1.3.4-dev (development of upcoming release)
 * Project: Renamed branch "master" to "main" and started "gitflow" branching model.
-* Fix bug: Add support for ChainerBackend class as keyring which iterates over all supported keyring backends (#1410)
+* Refactoring: Renamed qt4plugin.py to systrayiconplugin.py (we are using Qt5 for years now ;-)
+* Fix bug: Missing icon in SSH private key button (#1364)
+* Fix bug: Master issue for missing or empty system-tray icon (#1306)
+* Fix bug: System-tray icon missing or empty (GUI and cron) (#1236)
+* Fix bug: Improve KDE plasma icon compatibility (#1159)
 * Fix bug: Unit test fails on some machines due to warning "Ignoring XDG_SESSION_TYPE=wayland on Gnome..." (#1429)
 * Fix bug: Generation of config-manpage caused an error with Debian's Lintian (#1398).
 * Fix bug: Return empty list in smartRemove (#1392, Debian Bug Report 973760)
 * Breaking change: Minimal Python version 3.8 required (#1358).
 * Feature: Exclude /swapfile by default (#1053)
 * Documentation: Removed outdated docbook (#1345).
 * Build: Introduced .readthedocs.yaml as asked by ReadTheDocs.org (#1443).
+* Dependency: The oxygen icons should be installed with the BiT Qt GUI since they are used as fallback in case of missing icons
```

Comment:
This info should be added to the README.md. There it is where distro maintainers pick up this information.

Reply:
Yes, I forgot that :-)

I will add this ASAP

I would also like to keep it in the CHANGES since this is what pkg maintainers do read first I guess (not the diff of the README)